### DISCUSSION TITLE
How to obtain the extra pair exttaordinaire badges?

### DISCUSSION THREAD

**Discussion by dhans11** [community/community#138452] [Discussions] [answered]
### Select Topic Area

Question

### Body

what are the  steps or contributions required to achive it?

**Comment by vickybelario** [comment, marked as answer]
HI you can obtain this badge by [Coauthoring](https://docs.github.com/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) in a merged pull request. (Requires GitHub Desktop App)
